lib/puppet/transaction/resource_harness.rb

```diff
@@ -144,17 +144,17 @@ def sync_if_needed(param, context)
 
       event = create_change_event(param, current_value, historical_value)
       event.status = "failure"
-      event.message = param.format("change from %s to %s failed: #{detail}",
+      event.message = param.format(_("change from %s to %s failed: "),
```

Comment:
So I think I've narrowed the test failures down to these lines (and the ones like them below). I've gone through a number of iterations trying to make this syntax happy, but no luck so far. Thoughts?